### DISCUSSION TITLE
Whoogle LXC

### DISCUSSION THREAD

**Discussion by yodaphone** [tteck/Proxmox#500] [Help Needed] [answered]
Hi

can you please let me know where the whoogle.env or environment file is stored?

**Comment by chpego** [comment, marked as answer]
@yodaphone ,

In this repository, i think [here](https://github.com/tteck/Proxmox/blob/5a5afad573d651140cfcbd73757b886312208843/setup/whoogle-install.sh#L83) according with [this](https://github.com/benbusby/whoogle-search#systemd-configuration)

**Comment by TomBoehm** [comment]
in /usr/local/lib/python3.11/dist-packages/whoogle.env
and it has to be enabled in /etc/systemd/system/whoogle.service via 
[Service]
Environment=WHOOGLE_DOTENV=1
